test/test_get_env.cpp

```diff
@@ -49,4 +50,16 @@ TEST(TestGetEnv, test_get_env) {
 
 TEST(TestGetEnv, test_get_home) {
   EXPECT_STRNE(NULL, rcutils_get_home_dir());
+
+  const char * home = NULL;
+  const char * ret = rcutils_get_env("HOME", &home);
+  EXPECT_EQ(NULL, ret);
```

Comment:
@brawner mind to explain why can it be assumed that `HOME` is not in the environment?

Reply:
The return of `rcutils_get_env` is an error string. If it's null then the call succeeded. The actual variable value is assigned to `home`.

Reply:
Oops, nevermind.

Reply:
What is the point of this test though? Won't it fail if `HOME` isn't set on Windows, for example?

Reply:
The original intent of this check was to verify that "HOME" was defined, so that we can be sure that the expected post condition below was a result of setting each environment variable to NULL. However, I forgot to update this when I added in the #ifdef for windows.

I have since updated this to check the appropriate HOME/USERPROFILE variables in the ifdef below.